config/storage.yml

```diff
@@ -1,10 +1,14 @@
 test:
   service: Disk
   root: <%= Rails.root.join("tmp/storage") %>
+  google_api_token: <%= Rails.application.credentials.dig(:google_api_token) %>
```

Comment:
You lost me here... why do you store thee Google api key in a configuration file for storage?

Reply:
Leftovers from figuring out how to store it for the whole project.
removed.